pkg/handlers/aws/mutation/cni/calico/inject.go

```diff
@@ -38,21 +33,11 @@ type calicoPatchHandler struct {
 	variableFieldPath []string
 }
 
-var (
-	_ commonhandlers.Named     = &calicoPatchHandler{}
-	_ mutation.GeneratePatches = &calicoPatchHandler{}
-	_ mutation.MetaMutator     = &calicoPatchHandler{}
```

Comment:
we still want to keep validating `MetaMutator` interface.  `_ mutation.MetaMutator     = &calicoPatchHandler{}`

Reply:
I think should be covered by the `New...` func shouldn't it?

Reply:
Right this is not really needed, the code here fails to compile if `Mutate` is not implemented: https://github.com/d2iq-labs/capi-runtime-extensions/blob/8d4207bc721c275e233e914b4e21f84f35fc1d38/pkg/handlers/aws/mutation/metapatch_handler.go#L20-L36